app/views/nsm/steps/view_claim/allowed_work_items.html.erb

```diff
@@ -0,0 +1,113 @@
+<% title t('.page_title') %>
+
+<turbo-frame id="work_items">
```

Comment:
How come we're putting this in a turbo frame? I thought we'd just _removed_ all the turbo frames

Reply:
I would suggest that is a merge conflict, I will check and remove this

Reply:
removed as well the other place that had a turboframe